libp2p/network/url_conversion.go

```diff
@@ -127,7 +127,30 @@ func getPeerIP(hostname string) (net.IP, error) {
 		if peerIP == nil {
 			continue
 		}
+		// TECHDEBT: remove this log line once direction is clearer
+		// on supporting multiple network addresses per peer.
+		if len(addrs) > 1 {
+			logger.Global.Warn().
+				Array("resolved", newStringLogArrayMarshaler(addrs)).
+				IPAddr("using", peerIP)
```

Comment:
Can you add a message saying "resolved multiple addresses but only using one. See ticket #XXX for more details"